autopilot/hostfilter.go

```diff
@@ -87,6 +96,12 @@ func isOutOfFunds(cfg api.AutopilotConfig, h Host, c api.Contract) bool {
 	return c.RenterFunds().Cmp(sectorPrice.Mul64(3)) < 0 || percentRemaining < minContractFundUploadThreshold
 }
 
+func isOutOfCollateral(cfg api.AutopilotConfig, h Host, c api.Contract) bool {
+	collateral := calculateHostCollateral(cfg, *h.Settings)
+	percentRemaining, _ := big.NewRat(0, 1).SetFrac(c.HostCollateral().Big(), collateral.Big()).Float64()
```

Comment:
Instead of using `Float64` I'd do a `Cmp` against `big.NewRat(2, 100)`. Might be faster than doing the conversion.